src/Modules/ModulesGraded.jl

```diff
@@ -1098,20 +1098,20 @@ end
 ###############################################################################
 
 @doc raw"""
-    FreeMod_dec(R::CRing_dec, n::Int,VarName = :e; cached::Bool = false) 
+    FreeMod_dec(R::CRing_dec, n::Int, name::VarName = :e; cached::Bool = false) 
 
 Construct a decorated (graded or filtered) free module over the ring `R` with rank `n`
 with the standard degrees, that is the standard unit vectors have degree 0.
 Additionally one can provide names for the generators. If one does 
 not provide names for the generators, the standard names e_i are used for 
 the standard unit vectors.
 """
-function FreeMod_dec(R::CRing_dec, n::Int,VarName = :e; cached::Bool = false) 
+function FreeMod_dec(R::CRing_dec, n::Int, name::VarName = :e; cached::Bool = false) 
   return FreeMod_dec{elem_type(R)}(R, [Symbol("$name[$i]") for i=1:n], [decoration(R)[0] for i=1:n])
```

Comment:
Huh, wait, how did this code even work before, when `name` was not defined? Is it simply not exercised? Or does it perhaps accidentally reference the function `name` here when it does `$name`? Really weird...

Reply:
I thought the same, somehow this was syntactic correct julia code 😅
The point I spotted it was some weird printing when fiddling around with deepcopy.